scripts/installVNC.py

```diff
@@ -0,0 +1,34 @@
+from JumpScale9 import j
+from cloudbroker.data import Models
+
+
+def install(ip):
+    # create the vnc instance
+    url = 'http://{}:8091/vnc_auto.html?token='.format(ip)
```

Comment:
the url we register is supposed to be something secure and public not on 8091 probaby some proxy will be inbetween the actual app and the url we register.
so we need to be able to specificy something like 
http(s)://hostname.domain:portmaybe?/